### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by MadeOfClicks** [Gorgious56/asset_browser_utilities#82] [Ideas]

**Comment by Gorgious56** [comment]
Hello, I see your deleted comment, are you still looking for a solution ?

**Comment by MadeOfClicks** [comment]
Hey there,

one Bug Ive noticed:

External Files/Tags/Remove – Remove All activated

Trying to add Tags after that doesnt work, as the + / - signs are still locked.
I Have to go back into remove and uncheck Remove All to be able to add Tags again.

Best Regards

Simon Hernandez

Von: Simon ***@***.***>
Gesendet: Freitag, 1. Juli 2022 12:44
An: ***@***.***>
Betreff: AW: [Gorgious56/asset_browser_utilities] delete (Discussion #82)

Hey,

I thought it might have been an user error of mine, thats why Ive deleted untill I further tested your addon but as your asking, heres my process so far:


For Context:
Ive renamed a (procedural) Kit-Ops Material Pack (with Bulk Rename Utility) which contains individual .blend files for each material and came with previews (.png) for each file.
.blend files and .png files have identical names but the material names stayed of Course different.


So these things did not work as expected for me:

Batch Operations/External Files/Assets/Custom Operation   (same for External Folders)
Filter by Type/Materials
Custom Operation/Rename Material/Slot 0
Activated Same as Asset

I expected it to rename the materials to the name of the .blend file
I also did not see a change of the material name to the asset name containing the material

Batch Operations/External Files/Previews/Load from Disk   (same for External Folders)
Filter by Type/Materials

Preview Images dont load, only a grey missing Icon sign appears

I also tested this:
Batch Operations/Current File/Previews/Load from Disk
Filter by Type/Materials
Worked as expected, but its faster to go through the Cogwheel in the Asset Manager when doing it manually..


So far it is still an absolute timesaver and I wish had stumbled across it earlier..
Also I feel bad about using it for free, you should defintely think about naming a price for it in the future or atleast putting it on gumroad to let people grant you something for it even if the base price stays free 😉

Best Regards

Simon Hernandez

Von: ***@***.***>
Gesendet: Freitag, 1. Juli 2022 11:29
An: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Betreff: Re: [Gorgious56/asset_browser_utilities] delete (Discussion #82)


Hello, I see your deleted comment, are you still looking for a solution ?

**Comment by Gorgious56** [comment]
Thanks for the reports ! I fixed the first one
https://github.com/Gorgious56/asset_browser_utilities/issues/85

I'll make sure to go through the other ones :)

**Comment by Gorgious56** [comment]
You can now rename the assets as the blend file name in custom operations https://github.com/Gorgious56/asset_browser_utilities/commit/03f893825ce1391ae25ed81181f206e116eb4d77

**Comment by MadeOfClicks** [comment]
Hey,

I thought it might have been an user error of mine, thats why Ive deleted untill I further tested your addon but as your asking, heres my process so far:


For Context:
Ive renamed a (procedural) Kit-Ops Material Pack (with Bulk Rename Utility) which contains individual .blend files for each material and came with previews (.png) for each file.
.blend files and .png files have identical names but the material names stayed of Course different.


So these things did not work as expected for me:

Batch Operations/External Files/Assets/Custom Operation   (same for External Folders)
Filter by Type/Materials
Custom Operation/Rename Material/Slot 0
Activated Same as Asset

I expected it to rename the materials to the name of the .blend file
I also did not see a change of the material name to the asset name containing the material

Batch Operations/External Files/Previews/Load from Disk   (same for External Folders)
Filter by Type/Materials

Preview Images dont load, only a grey missing Icon sign appears

I also tested this:
Batch Operations/Current File/Previews/Load from Disk
Filter by Type/Materials
Worked as expected, but its faster to go through the Cogwheel in the Asset Manager when doing it manually..


So far it is still an absolute timesaver and I wish had stumbled across it earlier..
Also I feel bad about using it for free, you should defintely think about naming a price for it in the future or atleast putting it on gumroad to let people grant you something for it even if the base price stays free 😉

Best Regards

Simon Hernandez

Von: ***@***.***>
Gesendet: Freitag, 1. Juli 2022 11:29
An: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Betreff: Re: [Gorgious56/asset_browser_utilities] delete (Discussion #82)


Hello, I see your deleted comment, are you still looking for a solution ?
